src/pages/Account/Tabs/ModulesTab/Contract.tsx

```diff
@@ -289,9 +295,36 @@ function RunContractForm({
       onSubmit={onSubmit}
       result={
         connected ? (
-          <Button type="submit" variant="contained" sx={{maxWidth: "8rem"}}>
-            Write
-          </Button>
+          <Box>
+            <Button
+              type="submit"
+              disabled={transactionInProcess}
+              variant="contained"
+              sx={{width: "8rem", height: "3rem"}}
+            >
+              {transactionInProcess ? (
+                <CircularProgress size={30}></CircularProgress>
+              ) : (
+                "Write"
+              )}
+            </Button>
+            {!transactionInProcess && transactionResponse && (
+              <Alert
+                severity={
+                  transactionResponse?.transactionSubmitted
+                    ? "success"
+                    : "error"
+                }
+                sx={{
+                  overflowX: "auto",
```

Comment:
instead of letting ppl scroll on the x-axis, can we give it enough height to display all error messages? for easier accessibility

Reply:
updated the code and screenshot.